content/techniques/configuration.md

```diff
@@ -310,3 +312,31 @@ export class AppService {
   }
 }
 ```
+
+#### Expandable Variables
+
+The `@nestjs/config` package allows you to take advantage of the **"Expandable Variables"** functionality. You can use your exisiting `.env` variables to declare new variables within `.env` itself. For eg:
+
+```json
+APP_URL=mywebsite.com
+SUPPORT_EMAIL=support@${APP_URL}
+```
+
+> info **Hint** `@nestjs/config` package internally uses [dotenv-expand](https://github.com/motdotla/dotenv-expand).
+
+This can simply be done using the `expandVariables` _(boolean)_ option in the `ConfigModule` like this:
```

Comment:
Suggestion:

Replace this
"This can simply be done using the `expandVariables` _(boolean)_ option in the `ConfigModule` like this:"

with

"Enable environment variable expansion using the `expandVariables` property in the options object passed to the `forRoot()` method of the `ConfigModule`, as shown below:

Reply:
Done 👍